packages/scaffold-ui/src/partials/w3m-account-wallet-features-widget/index.ts

```diff
@@ -279,6 +270,23 @@ export class W3mAccountWalletFeaturesWidget extends LitElement {
     return html`<wui-balance dollars="0" pennies="00"></wui-balance>`
   }
 
+  private tabsTemplate() {
+    const isEVM = ChainController.state.activeChain === CommonConstantsUtil.CHAIN.EVM
+
+    if (!isEVM) {
+      return null
+    }
```

Comment:
nit: we can create a util `getTabsByNamespace` which would be more generic and respond empty (for now in btc/sol)